stories/_theme.scss

```diff
@@ -7,6 +7,7 @@
   --mdc-theme-error: #c30000;
   --mdc-required-input: #da1414;
   --progress-bar-color: #005cb9;
+  --main-content-height: unset;
```

Comment:
Does this need to be h-100 to retain compatibility?

Reply:
While it will not behave exactly the same, in most cases there isn't a noticeable difference. I don't think it should be 100% by default, but I also am not sure this warrants a breaking change. I am not willing to die on this hill though